.github/workflows/ci.yml

```diff
@@ -707,13 +662,9 @@ jobs:
     timeout-minutes: 130
     needs: build-pt
     steps:
-      - uses: actions/checkout@v3
+      - uses: trinodb/github-actions/checkout-and-setup@6a40cfbfa0ad484d69f5482b29c41fbaac72944e
```

Comment:
This is causing confusion since it's mixed with the "test with secrets" functionality. How much harder would it be to extract a commit?

Reply:
Also, any specific reason to keep actions used by a single repo (trinodb/trino) outside of the repo instead of composite actions at `.github/actions` probably?

Reply:
Done! The checkout actions need to be in a separate repo because this one is not checked out yet when this action is called.